content/docs/tasks/security/health-check/index.md

```diff
@@ -5,15 +5,11 @@ weight: 70
 keywords: [security,health-check]
 ---
 
-This task shows how to enable Kubernetes health checking for Citadel. Note this is an Alpha feature.
-
-Since Istio 0.6, Citadel has a health checking feature that can be optionally enabled.
-By default, the normal Istio deployment process does not enable this feature.
+Citadel has a health checking feature that can be optionally enabled.
 Currently, the health checking feature is able to detect the failures of Citadel CSR signing service,
```

Comment:
What does CSR stand for? Please expand abbreviations when you introduce them in a document. You can use the abbreviation on the following instances.
```suggestion
to detect the failures of the Citadel CSR signing service.
```